<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsProjectSurveySettingsWidgetBase</class>
 <widget class="QWidget" name="QgsProjectSurveySettingsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Survey Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QgsScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>504</width>
        <height>445</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QgsMessageBar" name="mMessageBar" native="true"/>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mKnownPointsGroupBox">
         <property name="title">
          <string>Known points</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelKnownPointsType">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mComboKnownPointsType"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QgsStackedWidget" name="mStackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="mPageNo">
             <layout class="QGridLayout" name="gridLayout" columnstretch="0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="mPageVector">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="labelVectorLayer">
                <property name="text">
                 <string>Layer with known points</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsMapLayerComboBox" name="mComboVectorLayer">
                <property name="showCrs">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Name attribute</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QgsFieldComboBox" name="mComboVectorField">
                <property name="filters">
                 <set>QgsFieldProxyModel::Int|QgsFieldProxyModel::LongLong|QgsFieldProxyModel::String</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>qgsstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QWidget</extends>
   <header>qgsmessagebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mComboKnownPointsType</tabstop>
  <tabstop>mComboVectorLayer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
